app/services/preserved_object_handler_results.rb

```diff
@@ -88,7 +89,8 @@ def result_hash(code, msg_args=nil)
   # result_array = [result1, result2]
   # result1 = {response_code => msg}
   # result2 = {response_code => msg}
-  def log_results
+  def report_results
```

Comment:
if `report_results` method returned the `result_array`, then we could end pohandler public methods with:

```
handler_results.report_results
```

instead of 
```
handler_results.report_results
handler_results.result_array
```